viral/learning_stages.py

```
@@ -22,112 +29,139 @@
+
+    pcs_mask, _ = get_place_cells(
+        session=session, spks=spks, rewarded=None, config=config, plot=True
     )
 
 
+def main() -> None:
+    # for mouse_name in SESSIONS_KEEP.keys():
+    #     mouse_sessions = get_mouse_sessions(mouse_name)
+
+    stages = ["unsupervised", "learning", "learned"]
+    result = {stage: None for stage in stages}
```

Ruff (0.14.0)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)